lumicks/pylake/tests/test_correlated_stack.py

```diff
@@ -419,6 +419,56 @@ def rgb_image_data_offset(spot_coordinates, warp_parameters):
     return img0, img, description, img_args["bit_depth"]
 
 
+def write_tiff_file(img0, img, description, bit_depth, n_frames, filename):
+    if bit_depth == 8:
+        img = img[:,:,0]
+        channels = 1
+    else:
+        channels = 3
+    movie = np.stack([img for n in range(n_frames)], axis=0)
+
+    tag_orientation = (274, 'H', 1, 1, False) # Orientation = ORIENTATION.TOPLEFT
+    tag_sample_format = (339, 'H', channels, (1, )*channels, False) # SampleFormat = SAMPLEFORMAT.UINT
+
+    with tifffile.TiffWriter(filename) as tif:
+        for n, frame in enumerate(movie):
+            str_datetime = f"{n*10+10}:{n*10+18}"
+            tag_datetime = (306, 's', len(str_datetime), str_datetime, False)
+            tif.save(frame, 
+                    description=json.dumps(description, indent=4),
+                    software="Bluelake Unknown",
+                    metadata=None, contiguous=False,
+                    extratags=(tag_orientation, tag_sample_format, tag_datetime))
+
+
+@pytest.fixture(scope="session")
+def rgb_file(tiff_dir, rgb_image_data):
```

Comment:
I would suggest referring to these fixtures as tiff files, or maybe just tiffs, so is very clear what kind of files they are.